controller:

  speed: 0.75
  rate: 30

control_output_topic: "/anicar/mux/ackermann_cmd_mux/input/path_follow"
path_topic: "/kal2/trajectory"